its/src/test/java/com/sonar/it/csharp/MetricsTest.java

```diff
@@ -305,38 +304,38 @@ public void complexityDistributionAtFileLevel() {
   public void linesOfCodeByLine() {
     String value = getFileMeasure("ncloc_data").getValue();
 
-    assertThat(value).contains("1=1");
-    assertThat(value).contains("2=1");
-    assertThat(value).contains("3=1");
-    assertThat(value).contains("4=1");
     assertThat(value).contains("5=1");
-
+    assertThat(value).contains("6=1");
+    assertThat(value).contains("7=1");
+    assertThat(value).contains("8=1");
     assertThat(value).contains("9=1");
-    assertThat(value).contains("10=1");
-    assertThat(value).contains("11=1");
-    assertThat(value).contains("12=1");
 
+    assertThat(value).contains("13=1");
+    assertThat(value).contains("14=1");
+    assertThat(value).contains("15=1");
     assertThat(value).contains("16=1");
-    assertThat(value).contains("17=1");
-    assertThat(value).contains("18=1");
-    assertThat(value).contains("19=1");
+
     assertThat(value).contains("20=1");
     assertThat(value).contains("21=1");
     assertThat(value).contains("22=1");
     assertThat(value).contains("23=1");
-
+    assertThat(value).contains("24=1");
     assertThat(value).contains("25=1");
     assertThat(value).contains("26=1");
     assertThat(value).contains("27=1");
-    assertThat(value).contains("28=1");
+
     assertThat(value).contains("29=1");
     assertThat(value).contains("30=1");
     assertThat(value).contains("31=1");
     assertThat(value).contains("32=1");
     assertThat(value).contains("33=1");
     assertThat(value).contains("34=1");
+    assertThat(value).contains("35=1");
+    assertThat(value).contains("36=1");
+    assertThat(value).contains("37=1");
+    assertThat(value).contains("38=1");
 
-    assertThat(value.length()).isEqualTo(143); // No other line
+    assertThat(value.length()).isEqualTo(144); // No other line
```

Comment:
this is because there's one value line with two-digit line number (as the header shifted the positions)